src/Data/Nat/Properties.agda

```diff
@@ -304,6 +304,10 @@ _>?_ = flip _<?_
 ≤∧≢⇒< {_} {suc n} (s≤s m≤n) 1+m≢1+n =
   s≤s (≤∧≢⇒< m≤n (1+m≢1+n ∘ cong suc))
 
+≢0⇒> :  ∀ {n} → n ≢ 0 → n > 0
+≢0⇒> {suc _} _    =  s≤s z≤n
+≢0⇒> {0}     0≢0 =  contradiction refl 0≢0
```

Comment:
Can we copy the name from `m<n⇒n≢0` which is the most similar property. i.e. `n≢0⇒n>0`

Reply:
Agreed.